core/snmp/api/src/main/java/org/opennms/netmgt/snmp/InetAddrUtils.java

```diff
@@ -31,6 +31,9 @@
 import java.net.InetAddress;
 import java.net.UnknownHostException;
 
+/**
+ * @deprecated Use {@link InetAddressUtils} instead.
+ */
 public abstract class InetAddrUtils {
```

Comment:
Can you please also add the `@Deprecated` annotation if it is deprecated. And it is also always nice to know why it was deprecated.